app/services/contracts/service.go

```diff
@@ -28,23 +28,39 @@ import (
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
-	abi "github.com/limechain/hedera-eth-bridge-validator/app/clients/ethereum/contracts/bridge"
+	routerAbi "github.com/limechain/hedera-eth-bridge-validator/app/clients/ethereum/contracts/router"
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	nilErc20Address = "0x0000000000000000000000000000000000000000"
+)
+
 type Service struct {
-	address    common.Address
-	contract   *abi.Bridge
-	Client     client.Ethereum
-	mutex      sync.Mutex
-	members    Members
-	serviceFee ServiceFee
-	logger     *log.Entry
+	address        common.Address
+	routerContract *routerAbi.Router
```

Comment:
since we are interacting with only one contract and the service is wrapping the `router` contract, lets just name the variable `contract`